src/Microsoft.AspNetCore.Mvc.Core/Internal/ObjectResultExecutor.cs

```diff
@@ -65,6 +66,11 @@ public class ObjectResultExecutor
         protected bool RespectBrowserAcceptHeader { get; }
 
         /// <summary>
+        /// Gets the value of <see cref="MvcOptions.ReturnHttpNotAcceptable"/>.
+        /// </summary>
+        protected bool ReturnHttpNotAcceptable { get; }
```

Comment:
Not for this PR but it's odd that `RespectBrowserAcceptHeader` and now `ReturnHttpNotAcceptable` are `protected` properties when both could be `private` fields. There's no user scenario for subclassing this `.Internal`-namespace type. @tuespetre you've been working w/ this code more recently that me. Am I missing anything?